packages/flx/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ homepage: https://github.com/flutter/flutter/tree/master/packages/flx
 dependencies:
   bignum: ^0.0.7
   asn1lib: ^0.4.1
-  cipher: ^0.7.1
+  pointycastle: ^0.10.0-dev
```

Comment:
We want just `0.10.0`. `-dev` is good if you need to consume a pre-release package.